doc/rls_mode/main.md

```diff
@@ -52,7 +52,7 @@ cd /path/to/rls
 rustup run nightly cargo install
 ```
 
-Because at the moment RLS links to the compiler and it assumes the compiler to be globally installed, one has to use rustup to start the `rls` (rustup will configure the environment accordingly):
+Because at the moment RLS links to the compiler and it assumes the compiler to be globally installed, one has to use rustup to start the `rls` (rustup will configure the environment accordingly). The following must be added to your VS Code `settings.json`:
```

Comment:
@redactedscribe,
Yes.
Please remove all changes in the file as it is inconsistent to all other sections.